pkg/cmd/clientConfig_test.go

```diff
@@ -0,0 +1,91 @@
+package cmd
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/kubernetes"
+	kFake "k8s.io/client-go/kubernetes/fake"
+)
+
+type args struct {
+	k8Client kubernetes.Interface
+}
+
+func TestNewClientConfigCmd(t *testing.T) {
+	k8Client := &kFake.Clientset{}
+	tests := []struct {
+		name string
+		args args
```

Comment:
I think we could just replace this with the kubernetes.Interface or a function that returns kubernetes.Interface. I assume the idea with the args here is that is can be expanded upon in the future?
Also while nothing is manipulating the k8client, it is a pointer and so if the test where extended in the future, you might endup with test order problems where one test is changing the behaviour of the client and the next test is using the same client. Overall I prefer to keep the behaviour locked down to the individual test case. I have found having a func 
``` func()kubernetes.Interface{}``` to be the best approach overall